src/KeyboardButton.js

```diff
@@ -17,10 +17,13 @@ export default class KeyboardButton extends PureComponent {
 	handleClick = () => this.props.onClick(this.props.value)
 
 	render() {
+	
+		let classes = this.props.classes || '';
```

Comment:
@WiaczeslawP sh*t. This has introduced a breaking change. The latest build on NPM does not have the above line of code at all - completely omitted. ` classes is not defined`